index.html

```diff
@@ -280,15 +282,32 @@
             <footer class="floe-footer">
                 <div class="floe-content row">
                     <div class="columns large-6 small-12">
-                        <div>
-                            <p>
-                                Floe is a project of the
-                                <a href="http://idrc.ocadu.ca">Inclusive Design Research Centre</a>, <a href="http://ocadu.ca">OCAD University</a>.<br/>
-                            </p>
-                            <p>
-                                <span class="floe-hewlett-logo">Funded by a grant from <a href="http://www.hewlett.org">The William and Flora Hewlett Foundation</a>.</span>
-                            </p>
-                        </div>
+                            <div class="row">
+                                <div class="idrc-logo-container columns large-5 medium-7 small-9">
+                                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 375.87 77.161"><title>IDRC Logo</title><g fill="#00A0CC"><path d="M21.701 12.809c6.365 10.428 14.345 23.502 19.237 34.619 2.625-3.069 6.783-7.631 10.449-11.338V10.161H20.086l1.615 2.648z"></path><path d="M42.243 50.534c1.541 3.866 2.62 7.419 3.018 10.443.454 3.457-.195 6.477-1.783 8.285-.965 1.102-2.21 1.684-3.598 1.684-1.236 0-2.506-.635-3.576-1.787-1.674-1.803-2.512-4.474-2.3-7.328.364-4.902 2.73-9.046 4.806-11.798-4.7-11.134-13.068-24.844-19.67-35.661-.885-1.45-1.746-2.86-2.565-4.211H1.387v65h50V40.393c-3.537 3.694-7.154 7.734-9.144 10.141z"></path><path d="M38.501 67.117c.568.612 1.097.829 1.378.829.514 0 .952-.217 1.344-.662.986-1.125 1.395-3.393 1.062-5.916-.307-2.34-1.064-5.046-2.152-7.997-1.489 2.304-2.887 5.306-3.138 8.682-.148 1.996.416 3.889 1.506 5.064z"></path><circle cx="56.887" cy="35.659" r="2.5"></circle><path d="M19.387 4.16a3.004 3.004 0 0 1-3.001 3.001 3 3 0 1 1 3.001-3.001z"></path>
+                                    <text x="70" y="35" font-family="Helvetica" font-size="37" font-weight="700">inclusive design</text>
+                                    <text x="80" y="72" font-family="Helvetica" font-size="37" font-weight="700">research centre</text>
```

Comment:
There is a case to be made for an SVG logo for keeping styles like `font-family` inline - it means the logo markup is a self-contained "instance" of the logo and can be used elsewhere without change in appearance. Moving some of the styles to the CSS will mean this self-containment goes away.

I don't have a strong feeling about this, but want to make the opposing case.